codec/av1.md

```diff
@@ -108,12 +106,28 @@ The `Block` timestamp is translated from the __[PresentationTime]__ without the
 
 AV1 stored in Matroska restricts the allowed variations among the `Sequence Header OBUs` contained in the `CodecPrivate` and also in-band in the `Blocks`. The changes are restricted to the changes that are allowed for the `Sequence Header OBUs` of a `CVS`, i.e. the contents of the `Sequence Header OBUs` MUST be bit-identical each time a `Sequence Header OBU` appears except for the contents of __[operating_parameters_info]__. Furthermore the dimensions of all output frames MUST be equal.
 
-The first `Sequence Header OBU` of a `CVS` is stored in the `CodecPrivate` of a `Track`.
 
-If a __[operating_parameters_info]__ exists in a `Sequence Header OBU` then each keyframe `Block` MUST contain a `Sequence Header OBU` before the `Frame Header OBUs`.
+# Seeking and Sequence Header Requirements
+
+`SimpleBlocks` with the `Keyframe` flag set and `Blocks` inside `BlockGroups` that do not contain any `ReferenceBlock` elements constitute the Random Access Points (RAP) of a Matroska `Track`.
+
+Upon random access to a RAP or upon starting playback from the beginning the player/demuxer MUST prepend the `Sequence Header OBU` contained in the `CodecPrivate` to the data of the `Block` where decoding starts after having reverted any potentially applying `ContentCompression`. Afterwards playback proceeds normally including consumption of any `Sequence Header OBUs` that are found in the bitstream.

```

Comment:
It's obvious to you and me, but is it really obvious to everyone? Image a track using header removal compression that applies to the data of the (Simple)Blocks. Does really everybody know in which order the prepending of the `Sequence Header OBUs` and the prepending of whatever should be prepended to the data should happen? I am not so sure and in order to avoid confusion I inserted this clause.
I did indeed not think about the scope of the `ContentCompression`, but now that I think about it my wording ("any potentially applying") fits even when taking this into account.

Reply:
We're not going to explain how each Matroska element works (like DisplayWidth/Height) because this is a new document. IMO `ContentCompression` is completely orthogonal to this spec and should not be taken in consideration at all. If that happens it's outside of this spec and transparent.
We also don't define what should be put in `Cues` even though it has an impact on seeking. (but now that I think of it we might, because Switch frames can do good Cue points even though they are not keyframes (AFAIK)).